environment/deployments/panda/cloudsql/main.tf

```diff
@@ -9,6 +9,10 @@ module "private-postgres" {
       "name" : "science-platform-int external-nat-ip",
       "value" : "35.239.140.105/32"
     },
+    {
+      "name" : "panda-dev moderatemem-external-nat-ip",
```

Comment:
The name here seems to suggest it only adds one of five clusters. If it allows all of them,  maybe just don't specify which cluster (e.g. "panda-dev external-nat-ip").  If each cluster needs to be specified individually, I think we need the "merge" cluster the most for this connectivity.